zone/embparser_api.cpp

```diff
@@ -3744,7 +3744,7 @@ void perl_register_quest()
 	package.add("clear_zone_flag", &Perl__clear_zone_flag);
 	package.add("clearspawntimers", &Perl__clearspawntimers);
 	package.add("collectitems", &Perl__collectitems);
-	package.add("commify", &Perl__commify);
+	package.add("Commify", &Perl__commify);
```

Comment:
This is case sensitive so need to undo this change to prevent api breakage

Reply:
Good eye, I had to revert a handful of these already as the IDE mucked up one of my refactors